vsintegration/src/unittests/Tests.LanguageService.Completion.fs

```diff
@@ -474,6 +474,75 @@ type AutoCompletionListTests() as this  =
                 "type B() = inherit A(System.String.)"
             ]
         AssertCtrlSpaceCompleteContains code "System.String." ["Empty"] ["Array"; "Collections"]
+    
+    [<Test>]
+    [<Category("Completion in object initializer")>]
+    member public this.``ObjectInitializer.CompletionForProperties``() =
+        let typeDef1 = 
+            [
+                "type A() = "
+                "   member val SettableProperty = 1 with get,set"
+                "   member val AnotherSettableProperty = 1 with get,set"
+                "   member val NonSettableProperty = 1"
+            ]
+        AssertCtrlSpaceCompleteContains (typeDef1 @ ["A((**))"]) "A((**)" ["SettableProperty"; "AnotherSettableProperty"] ["NonSettableProperty"] 
+        AssertCtrlSpaceCompleteContains (typeDef1 @ ["A(S = 1)"]) "A(S" ["SettableProperty"] ["NonSettableProperty"] 
+        AssertCtrlSpaceCompleteContains (typeDef1 @ ["A(S = 1)"]) "A(S = 1" [] ["SettableProperty"; "NonSettableProperty"] // neg test
+        AssertCtrlSpaceCompleteContains (typeDef1 @ ["A(S = 1,)"]) "A(S = 1," ["AnotherSettableProperty"] ["NonSettableProperty"] 
+        AssertCtrlSpaceCompleteContains (typeDef1 @ ["new A((**))"]) "A((**)" ["SettableProperty"; "AnotherSettableProperty"] ["NonSettableProperty"] 
+        AssertCtrlSpaceCompleteContains (typeDef1 @ ["new A(S = 1)"]) "A(S" ["SettableProperty"] ["NonSettableProperty"] 
+        AssertCtrlSpaceCompleteContains (typeDef1 @ ["new A(S = 1)"]) "A(S = 1" [] ["SettableProperty"; "NonSettableProperty"] // neg test
+        AssertCtrlSpaceCompleteContains (typeDef1 @ ["new A(S = 1,)"]) "A(S = 1," ["AnotherSettableProperty"] ["NonSettableProperty"] 
+
+        let typeDef2 = 
+            [
+                "type A<'a>() = "
+                "   member val SettableProperty = 1 with get,set"
+                "   member val AnotherSettableProperty = 1 with get,set"
+                "   member val NonSettableProperty = 1"
```

Comment:
Hmmm... In theory we should handle bethod calls as well, since settable properties can be used with the return result of any method call, not just constructors.